apps/website/src/pages/docs/header.mdx

```diff
@@ -14,68 +14,49 @@ import * as AllExamples from 'examples';
 </LiveExample>
 
-The headers seen when in our CONNECT Portal environment. A header sits at the top of a page and is part of a consistent user experience.
-
-## Main CONNECT Portal header
-
-The CONNECT portal header contains the services menu, home, enterprise, assets and projects links beginning from left to right. From right to left there is the user profile, help, and notification links.
+The first thing most people look at when they arrive at a new web page is the header. It is usually stuck at the top of the page. The header contains vital information about your company's name and function, as well as the purpose of the specific page.
 
-Header with circular callouts, numbered from 1 to 8.
+## Usage
 
-- Services menu
-- Home page
-- Enterprise portal
-- Assets menu
-- Projects menu
-- Notifications
-- Help
-- User icon
+`Header` consists of various subcomponents and can be customized by modifying its props.
 
-### Actions
+- Logo can be added using the `appLogo` prop and specifying the desired logo under the `<HeaderLogo>` subcomponent.
+- Breadcrumbs trail can be implemented using the `breadcrumbs` prop and `<HeaderBreadcrumbs>` subcomponent. Individual buttons can be added within the breadcrumbs trail using the [`<HeaderButton>` subcomponent](#headerbutton-subcomponent).
+- The `actions` prop can be used to add any arbitrary buttons and content near the right end of the header.
+- The `menuItems` prop can be used to show any additional actions inside an overflow menu located to the right of `actions`.
 
-- Services menu pops up a list of services available to the user.
-- Home page takes you to CONNECT Center.
-- Enterprise Portal takes you to the Subscription Services Portal.
-- Assets menu takes you to the assets page.
-- Projects menu takes you to the ProjectWise Projects page.
-- Notifications pops up your list of notifications.
-- Help takes you to Bentley Communities for Bentley Cloud and Web Services.
-- User profile is a drop down with options for your user profile.
+### `<HeaderButton>` subcomponent
 
-## Secondary header
+`HeaderButton` is designed to offer a range of behaviors that are dependent on the props that are passed to it.
 
-The secondary header contains the product icon, product name, iModel name and the name of the initially opened view. On the right side of the bar is the settings menu.
+- `onClick` prop: behaves like a standard button.
+- `as='a'` and `href` props: behaves like a regular anchor link.
+- `menuItems` prop: behaves like a dropdown button.
+- Both `menuItems` and `onClick`/`href` props: behaves like a split button, where the left part is the main button/link and the right part opens a dropdown menu.
 
-Secondary header with circular callouts, numbered from 1 to 5.
+The `isActive` prop can be used to indicate the current breadcrumb.
 
-- Product icon
-- Product name
-- iModel name
-- View name
-- Settings
-
-### Actions
+<LiveExample src='Header.full.tsx'>
```

Comment:
this example could be moved above the HeaderButton section, so that is sits closer to the general usage bullet points